src/cockpit/channels/filesystem.py

```diff
@@ -191,6 +192,11 @@ def do_data(self, data):
 
             try:
                 self._tempfile = os.fdopen(fd, 'wb')
+                try:
+                    shutil.copystat(self._path, self._temppath)
```

Comment:
This is crap actually. It doesn't copy the owner/group (which we do want), and does copy the atime/mtime (which we don't need).